package.json

```diff
@@ -22,7 +22,8 @@
     "request-promise": "^4.1.1",
     "ts-node": "^3.1.0",
     "tslint": "^5.4.3",
-    "typescript": "^2.4.1"
+    "typescript": "^2.4.1",
+    "strong-docs": "git://github.com/strongloop/strong-docs.git#master"
```

Comment:
Let's release a new version 1.4.0 of strong-docs before landing this patch, so that we can use regular `"^1.4.0"` version spec here.